docs/en/operations/workload-scheduling.md

```diff
@@ -79,10 +79,13 @@ graph TD
 
 **Possible node types:**
 * `inflight_limit` (constraint) - blocks if either number of concurrent in-flight requests exceeds `max_requests`, or their total cost exceeds `max_cost`; must have a single child.
+* `bandwidth_limit` (constraint) - blocks if burst exceeds `max_burst` (default 0) or current bandwidth exceeds `max_speed` (0 means unlimited); must have a single child.
 * `fair` (policy) - selects the next request to serve from one of its children nodes according to max-min fairness; children nodes can specify `weight` (default is 1).
 * `priority` (policy) - selects the next request to serve from one of its children nodes according to static priorities (lower value means higher priority); children nodes can specify `priority` (default is 0).
 * `fifo` (queue) - leaf of the hierarchy capable of holding requests that exceed resource capacity.
 
+To be able to use the full capacity of the underlying resource, you should use `inflight_limit`. Note that a low number of `max_requests` or `max_cost` could lead to not full resource utilization, while too high numbers could lead to empty queues inside the scheduler, which in turn will result in policies being ignored (unfairness or ignoring of priorities) in the subtree. On the other hand, if you want to protect resources from too high utilization, you should use `bandwidth_limit`. It throttles when the amount of resource consumed in `duration` seconds exceeds `max_burst + max_speed * duration` bytes. Two `bandwidth_limit` nodes on the same resource could be used to limit peak bandwidth during short intervals and average bandwidth for longer ones.
```

Comment:
Is true that since we can't reset `max_burst` ? In that case it looks for me not like peak/average, but like we can decrease speed over the time.

Reply:
> Is true that since we can't reset max_burst ?

What do you mean? What is true?

> If you are talking about avg+peak limit it is not completely covered here, because it is pretty complex usage pattern, probably worth it own section in docs. But there is a test for it. It is configured with the following two nodes:

```cpp 
    // Burst = 100 token
    // Peek speed = 50 token/s for 10 seconds (50 * 10 = 5000 tokens = burst of "/avg" bucket)
    // Avg speed = 10 tokens/s afterwards
    t.add<ThrottlerConstraint>("/", "<max_burst>100.0</max_burst><max_speed>50.0</max_speed>");
    t.add<ThrottlerConstraint>("/avg", "<max_burst>5000.0</max_burst><max_speed>10.0</max_speed>");
```